Feature/header

omerfarukorc opened the pull request:
Headerdaki toggle menü kaldırıldı Giriş Yap ve Kayıt Ol Headera eklendi.
Görsel iyileştirmeler yapıldı.
Font boyutu ve Heeader konumu değiştirildi.
Header rengi değiştirildi

omerfarukorc committed:
Header bileşeni güncellendi

omerfarukorc committed:
Header ve App dosyalarındaki düzenlemeler

mehmethayrullahozkul commented on src/App.tsx:
```diff
@@ -14,24 +14,10 @@ import Support from "./pages/Support";
   return (
-    <ThemeProvider theme={isDarkMode ? darkTheme : lightTheme}>
-      <Header
-        currentTheme={isDarkMode ? "dark" : "light"}
-        toggleTheme={toggleTheme}
-      />
+    <>
+      <Header />
```
Uygulamadan temayı neden çıkarttık tamamen?

omerfarukorc replied:
Build hatalarına neden oluyordu. implementasyonu tam olarak hazır değil .Bence şu anda projenin temel fonksiyonlarına (halı saha rezervasyonu vs odaklanmamız gerekiyor

mehmethayrullahozkul replied:
Tamam da çalışan bir yapı varken yeni değişiklikler yüzünden build hatalarına neden oluyor demek doğru değil sanki...

mehmethayrullahozkul commented on src/css/header/header.css:
```diff
@@ -29,26 +29,36 @@ body {
+    padding-right: 2rem;
+}
+
+.header-tab-left {
+    display: flex;
+    gap: 20px;
+    margin-left: 0;
 }
```
Piksel gapleri sx üzerinden React kullanarak verebiliriz.

mehmethayrullahozkul commented on src/css/header/header.css:
```diff
@@ -29,26 +29,36 @@ body {
-    padding: 1rem;
+    padding: 0 0.5rem;
     border-radius: 2rem;
+    margin: 0 15px;
     
```
Px gap

mehmethayrullahozkul commented on src/css/header/header.css:
```diff
@@ -356,4 +389,24 @@ body {
\ No newline at end of file
+}
+
+.login-button {
+    background-color: #2a952e;
+    border-radius: 20px;
+    padding: 8px 16px;
+}
+
```
CSS üzerinden px vermek yerine React sx kullanabiliriz

mehmethayrullahozkul commented on src/components/Header.tsx:
```diff
@@ -1,28 +1,14 @@
-    } else {
-      setIsAtTop(false);
-    }
+    setIsAtTop(window.scrollY === 0);
```
Zeki adamın kodu bi başka oluyo

mehmethayrullahozkul reviewed (commented):
eline sağlık onay verim mi

mehmethayrullahozkul reviewed (changes requested):
tema tamamen kaldırılmamalı

omerfarukorc committed:
Add theme toggle functionality and minimize theme button

- Re-implement theme toggle feature
- Add minimal theme button styling
- Update header component with theme props
- Add theme-related CSS styles

mehmethayrullahozkul reviewed (approved):
gj gn
